src/ORM/DataList.php

```diff
@@ -1691,7 +1671,7 @@ public function first()
      * The object returned is not cached, unlike {@link DataObject::get_one()}
      * @return T|null
      */
-    public function last()
+    public function last(): mixed
```

Comment:
How come this is mixed, but `first()` is specific? Looks like it should be:
```suggestion
    public function last(): ?DataObject
```

Reply:
Updated